Utilities/Logging.cs

```diff
@@ -108,32 +108,35 @@ public static void incrementLogs()
         {
             // Obtain files, sorting by last write time to ensure that older files are incremented prior to newer files
             DirectoryInfo directoryInfo = new DirectoryInfo(Constants.DATA_DIR);
-            foreach (FileInfo file in directoryInfo.GetFiles().OrderBy(f => f.LastWriteTimeUtc).ToList())
+            if (directoryInfo.Exists)
```

Comment:
Shouldn't we be creating the dir if it doesn't exist?

Reply:
The installer probably should do this, but it's not sufficient since EDDI would still crash as soon as someone deleted the folder to clear config files and then tried to restart EDDI.